lib/test/tasks/create_db_copies.rb

```diff
@@ -5,8 +5,10 @@ def run
     # The commands below will error if there are any active connections to the
     # database, so disconnect.
     ActiveRecord::Base.connection_pool.connections.each(&:disconnect!)
+    # Give a little time for the disconnections to complete (?).
+    sleep(0.5)
```

Comment:
See this commit message for an explanation of this addition: https://github.com/davidrunger/david_runger/pull/6226/commits/fbbc66f2b9b9807e1107ecdfcacc5bf6e8931243 (which I think might actually not be really related to this change?, even though it seems super suspicious that I was already changing this file on the lines right below here).